src/lualib/Symbol.ts

```diff
@@ -1,3 +1,19 @@
+declare function setmetatable<T extends object>(obj: T, metatable: any): T;
+
+const symbolMetatable = {
```

Comment:
This should probably have leading underscores (we usually use 4) to prevent name collisions.

Reply:
Oh, right. They compile to locals, so it's not a problem in bundle, but that might cause issues with inlining.